Sub/Src/ComputerVision/Nodes/CamNode.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+import cv2
+import math
+import random
+import socket
+import sys
+import struct
+import math
+import io
+import time
+from ctypes import *
+from libs.darknet import *
+from MechOS.message_passing.Nodes.node_base import node_base
+
+class CamNode(node_base):
+
```

Comment:
Please give a class description here. Refer to the software documentation manual in the wiki on how to.